src/test/java/com/crawler/extractor/CrawlerConfServiceTest.java

```diff
@@ -0,0 +1,104 @@
+package com.crawler.extractor;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.webAppContextSetup;
+
+import java.nio.charset.Charset;
+import java.util.Date;
+
+import org.bson.types.ObjectId;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.embedded.LocalServerPort;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.context.WebApplicationContext;
+
+import com.crawler.extractor.model.Crawler;
+import com.crawler.extractor.model.CrawlerConf;
+import com.crawler.extractor.model.Status;
+import com.crawler.extractor.repository.ICrawlerConfRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class CrawlerConfServiceTest {
+	
+	private final String URL_TO_UPDATE_CRAWLER_CONF = "extractor/rest/v1/crawler/conf";
+	
+	@LocalServerPort
+	private int port;
+
+	private MediaType contentType = new MediaType(MediaType.APPLICATION_JSON.getType(),
+			MediaType.APPLICATION_JSON.getSubtype(), Charset.forName("utf8"));
+
+	private MockMvc mockMvc;
+
+	private CrawlerConf crawlerConf;
```

Comment:
it is not the best idea to have `crawlerConf` as class variable, remove it and create new `CrawlerConf` for each test

Reply:
But if i will create new CrawlerConf for each test it will be copy paste.
I can create method, that will return CrawlerConf, and use it for each test, but i don't sure that it will be right.